From 76764e1e80559bfbe888903b0857816fb7d239ec Mon Sep 17 00:00:00 2001
From: Athenas Jimenez Gonzalez <athenas.jimenez.gonzalez@intel.com>
Date: Tue, 16 Jan 2018 00:56:37 +0000
Subject: [PATCH] Disable R manual and tests

---
 dev/make-distribution.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dev/make-distribution.sh b/dev/make-distribution.sh
index 48a8244..9124057 100755
--- a/dev/make-distribution.sh
+++ b/dev/make-distribution.sh
@@ -221,7 +221,7 @@ if [ "$MAKE_R" == "true" ]; then
   pushd "$SPARK_HOME/R" > /dev/null
   # Build source package and run full checks
   # Do not source the check-cran.sh - it should be run from where it is for it to set SPARK_HOME
-  NO_TESTS=1 "$SPARK_HOME/R/check-cran.sh"
+  NO_TESTS=0 NO_MANUAL=0 "$SPARK_HOME/R/check-cran.sh"
 
   # Move R source package to match the Spark release version if the versions are not the same.
   # NOTE(shivaram): `mv` throws an error on Linux if source and destination are same file
-- 
2.15.1

