From 19d21c89a1baad4fd95cd7831dd305702581dec0 Mon Sep 17 00:00:00 2001
From: Athenas Jimenez <athenas.jimenez.gonzalez@intel.com>
Date: Wed, 31 Jan 2018 17:57:34 +0000
Subject: [PATCH] Disable Zinc installation

---
 build/mvn | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/mvn b/build/mvn
index 1e393c3..cc9ab0d 100755
--- a/build/mvn
+++ b/build/mvn
@@ -131,7 +131,7 @@ if [ "$1" == "--force" ]; then
 fi
 
 # Install the proper version of Scala, Zinc and Maven for the build
-install_zinc
+#install_zinc
 install_scala
 install_mvn
 
-- 
2.16.1

